### ISSUE TITLE
'active' not recognized on CSV import

### ISSUE THREAD

**Issue by Traumflug** [thirtybees/thirtybees#252]
Apparently CSV import doesn't work as well as it should. Eezee provided screenshots as well as a test file:

![](http://i.imgur.com/cvURzx7.png)

![](http://i.imgur.com/LOHJTyp.png)

Test file: https://forum.thirtybees.com/assets/uploads/files/1496405916691-tb_cat.xls

Discussion: https://forum.thirtybees.com/topic/363/csv-import-categories-bug

**Comment by Traumflug** [comment]
@firstred fixed this issue and @jnsgioia confirmed the fix to be working. Excellent, thank you!

**Comment by alwayspaws** [comment]
Tested one product with combinations to be shown in three categories. I tried openoffice ods and then csv formats.

Categories are still a problem:  (tags work perfectly, btw).

1.  Quotes around the category name and a space in between each category. Does not work.
2.  Comma and space after each category. Does not work. 
3.  The only way it works is with a semicolon and space in between each category. 

If item #3 is the way to go, I'll use it, but I thought we standardized everything to use commas.

The screenshots explain this better. Also, I put three screenshots together to show that even though there were quotes around all three categories in the csv, the first category did not have any quotes and all three were strung together and created as a new category.

I tried to attach the spreadsheets here but it wasn't accepted.

This is a compilation of three screenshots put together:

![categories do not work with or without quotation marks via ods or csv - tags work with commas and spaces](https://user-images.githubusercontent.com/26662422/28749749-f5309650-749f-11e7-8e96-9d3902451d00.png)


and this one shows the results of the tests:


![csv categories imports - tests](https://user-images.githubusercontent.com/26662422/28749759-5c686a46-74a0-11e7-8c10-f6a1ab346749.png)
